tests/countries/test_cambodia.py

```diff
@@ -0,0 +1,563 @@
+        self.assertNoHolidayName(name, 2017)
+        self.assertHolidaysName(
+            name, (f"{year}-05-20" for year in range(2018, 2020))
+        )
+        self.assertNoHolidayName(name, 2020)
+
+    def test_international_children_day(self):
+        name = "ទិវាកុមារអន្តរជាតិ"
+        self.assertHolidaysName(
+            name,
+            (f"{year}-06-01" for year in range(1993, 2020)),
+        )
+        self.assertNoHolidayName(name, 2020)
+
+    def test_queen_mother_monineath_birthday(self):
+        name = (
+            "ព្រះរាជពិធីបុណ្យចម្រើនព្រះជន្ម សម្តេចព្រះមហាក្សត្រី "
+            "ព្រះវររាជមាតា នរោត្តម មុនិនាថ​ សីហនុ"
+        )
+        self.assertNoHolidayName(name, 1993)
+        self.assertHolidaysName(
+            name, (f"{year}-06-18" for year in range(1994, 2058))
+        )
+
+    def test_constitution_day(self):
+        self.assertHolidaysName(
+            "ទិវាប្រកាសរដ្ឋធម្មនុញ្ញ",
+            (f"{year}-09-24" for year in range(1993, 2058)),
+        )
+
+    def test_king_sihanouk_memorial_day(self):
+        name = (
+            "ទិវាប្រារព្ឋពិធីគោរពព្រះវិញ្ញាណក្ខន្ឋ ព្រះករុណា ព្រះបាទសម្តេចព្រះ"
+            " នរោត្តម សីហនុ ព្រះមហាវីរក្សត្រ ព្រះវររាជបិតាឯករាជ្យ បូរណភាពទឹកដី"
+            " និងឯកភាពជាតិខ្មែរ ព្រះបរមរតនកោដ្ឋ"
+        )
+        self.assertNoHolidayName(name, 2011)
+        self.assertHolidaysName(
+            name,
+            (f"{year}-10-15" for year in range(2012, 2058)),
+        )
+
+    def test_paris_peace_agreement_day(self):
+        name = "ទិវារំលឹកសន្ធិសញ្ញាសន្តិភាពទីក្រុងប៉ារីស"
+        self.assertHolidaysName(
+            name,
+            (f"{year}-10-23" for year in range(1993, 2020)),
+        )
+        self.assertNoHolidayName(name, 2020)
+
+    def test_king_sihamoni_coronation_day(self):
+        name = (
+            "ព្រះរាជពិធីគ្រងព្រះបរមរាជសម្បត្តិ របស់ ព្រះករុណា "
+            "ព្រះបាទសម្តេចព្រះបរមនាថ នរោត្តម សីហមុនី "
+            "ព្រះមហាក្សត្រនៃព្រះរាជាណាចក្រកម្ពុជា"
+        )
+        self.assertNoHolidayName(name, 2003)
+        self.assertHolidaysName(
+            name, (f"{year}-10-29" for year in range(2004, 2058))
+        )
+
+    def test_national_independence_day(self):
+        self.assertHolidaysName(
+            "ពិធីបុណ្យឯករាជ្យជាតិ",
+            (f"{year}-11-09" for year in range(1993, 2058)),
+        )
+
+    def test_international_human_rights_day(self):
+        name = "ទិវាសិទ្ធិមនុស្សអន្តរជាតិ"
+        self.assertHolidaysName(
+            name,
+            (f"{year}-12-10" for year in range(1993, 2020)),
+        )
+        self.assertNoHolidayName(name, 2020)
+
+    def test_meak_bochea(self):
+        name = "ពិធីបុណ្យមាឃបូជា"
+        dt = (
+            "2015-02-03",
+            "2016-02-22",
+            "2017-02-11",
+            "2018-01-31",
+            "2019-02-19",
+        )
+        self.assertHolidaysName(name, dt)
+        self.assertNoHolidayName(name, Cambodia(years=2058))
```

Comment:
```suggestion
        self.assertHolidaysName(
            name,
            "2015-02-03",
            "2016-02-22",
            "2017-02-11",
            "2018-01-31",
            "2019-02-19",
        )
        self.assertNoHolidayName(name, 2058)
```
And same for other similar holidays below.

Reply:
I'm basing this on my old code for Thailand's testcase - maybe I should get back to that later when we're done with this 🙏